src/rcore_web.c

```diff
@@ -682,11 +683,13 @@ void SetMouseCursor(int cursor)
         {
             TRACELOG(LOG_WARNING, "Cursor value out of bound (%d). Setting to default", cursor);
```

Comment:
@raysan5 From your experience, do you know any scenario where `SetMouseCursor()` would normally be called during the drawing step? If yes, then I guess this TraceLog should be omitted (similar case: https://github.com/raysan5/raylib/issues/3366#issuecomment-1743836669).

Reply:
@ubkp Yes, it can be called in the update/drawing step, for example, when mouse is over an screen element to select it and cursor needs to be changed. In those cases, the function should also avoid changing cursor if requested one is already the current one.